src/DataDryad.jl

```diff
@@ -0,0 +1,42 @@
+immutable DataDryad <: DataRepo
+end
+
+base_url(::DataDryad) = "https://datadryad.org/mn/"
+
+function description(::DataDryad, mainpage)
+    desc = replace(text_only(first(matchall(sel".article-abstract", mainpage.root))), "Abstract ", "")
+    author = text_only(first(matchall(sel".pub-authors a", mainpage.root)))
+    license = getattr(first(matchall(sel".single-image-link", mainpage.root)), "href")
+    date = now()
+    paper = text_only(first(matchall(sel".citation-sample a", mainpage.root)))
+    dataset = replace(text_only(last(matchall(sel".publication-header p", mainpage.root))), "DOI: ", "")
+
+    final = escape_multiline_string("""
+    Author: $(author) et. al.
+    License: $(license)
+    Date: $(date)
+
+    $(desc)
+
+    Please cite this paper:
+    $(paper)
+    as well as this dataset:
+    $(dataset)
+    if you use this in your research.
+    """, "\$")
+end
+
+function get_urls(repo::DataDryad, page)
+    urls = []
+    paper = text_only(last(matchall(sel".publication-header p", page.root)))
+    download_url = replace(paper, "DOI: https://", "https://datadryad.org/mn/object/http://dx.") * "/1/bitstream"
```

Comment:
This isn't going to work for datasets with multiple files
e.g. https://datadryad.org/resource/doi:10.5061/dryad.f9s4424

Probably good to add that one as an extra text

Reply:
Implemented.